crates/interpreter/src/valid.rs

```diff
@@ -885,7 +888,12 @@ impl<'a, 'm> Expr<'a, 'm> {
         let n = self.labels.len();
         check(l < n)?;
         let source = self.branch_source();
+        // Apply the side table entry in flash and compute the expected target branch.
```

Comment:
Yes, but we shouldn't use the word flash. We're reading the side table from the side table section the input binary slice.